pkg/secrethub/service.go

```diff
@@ -3,12 +3,13 @@ package secrethub
 import (
 	"github.com/secrethub/secrethub-go/internals/api"
 	"github.com/secrethub/secrethub-go/internals/errio"
+	"github.com/secrethub/secrethub-go/pkg/secrethub/credentials"
 )
 
 // ServiceService handles operations on service accounts from SecretHub.
 type ServiceService interface {
 	// Create creates a new service account for the given repo.
-	Create(path string, description string, verifier Verifier, encrypter Encrypter) (*api.Service, error)
+	Create(path string, description string, verifier credentials.Verifier, encrypter credentials.Encrypter) (*api.Service, error)
```

Comment:
I think we discussed this, but this signature is completely unclear to the user. They won't know where to start when they want to call this function.